pxr/usd/sdf/wrapPredicateExpression.cpp

```diff
@@ -50,7 +50,12 @@ _Repr(SdfPredicateExpression const &self) {
             + TfPyRepr(self.GetText()) + ")";
     }
 }
-                           
+
+static std::string const&
+_GetParseError(SdfPredicateExpression const &self) {
+    return self.GetParseError();
```

Comment:
I noticed one other thing while looking into this that didn't have an affect on the Boost Python wrapping issue introduced with C++20 but might still be of interest:

`predicateExpression.h` has two ref-qualified versions of `SdfPredicateExpression::GetParseError()`:
```C++
std::string const &GetParseError() const &;
std::string GetParseError() const &&;
```

For both of these though, the body is just:
https://github.com/PixarAnimationStudios/OpenUSD/blob/6e34a3338d666a2c272ad0e1ed658d61e776df83/pxr/usd/sdf/predicateExpression.h#L268

Compare that to a similar pattern in `assetPath.h` for `SdfAssetPath`'s `GetAssetPath()/GetResolvedPath()` where the rvalue ref-qualified versions do a `std::move()`:
https://github.com/PixarAnimationStudios/OpenUSD/blob/6e34a3338d666a2c272ad0e1ed658d61e776df83/pxr/usd/sdf/assetPath.h#L136

Maybe `return std::move(_parseError)` was intended in the rvalue ref-qualified version of `SdfPredicateExpression::GetParseError()` as well?

The `SdfAssetPath` functions don't suffer from the same C++20-related issue as `SdfPredicateExpression::GetParseError()` since they're already wrapped using static functions.

Reply:
FYI this is addressed in an upcoming change.